README.md

```diff
@@ -13,6 +13,8 @@ A list of resources connected to the [SpaceTraders API](https://spacetraders.io/
 
 ## Client
 
+* [Deliverance](https://github.com/Stumblinbear/Deliverance) - VueJS-based web interface for Space Traders
```

Comment:
Please end the line with a period "." as per the the [contribution guidelines](https://github.com/SpaceTradersAPI/awesome-spacetraders/blob/main/contributing.md).